outlook/src/taskpane/components/Contact/ContactPage/ContactPage.tsx

```diff
@@ -97,9 +97,13 @@ class ContactPage extends React.Component<ContactPageProps, ContactPageState> {
     }
 
     private onPartnerChanged = (partner: Partner) => {
+        this.setState({partner: partner});
         this.props.onPartnerChanged(partner);
     }
 
+    private onCompanyUpdated = () => {
+        this.fetchContact();
```

Comment:
So when the company is updated (meaning enriched I suppose?), you send another request to fetch it again?
That doesn't seem right, what am I missing?

(Method could use a bit of doscstring, when does it happen, why do we need to fetch the contact, ...)

Reply:
That is correct.

After taking a deeper look at this, this has now been changed as it appears that we don't need it, since we can directly update the partner using the company without fetching it again.